src/api/wallet.ts

```diff
@@ -44,7 +42,7 @@ export const connectToWallet = async (): Promise<boolean> => {
 
 export const getAccountAddress: () => Promise<string | null> = async () => {
   try {
-    const data = await window.aptos.account();
+    const data = await window.aptos?.account();
```

Comment:
The linked Sentry issue includes an error log `window.aptos.account is not a function` implying that `window.aptos` is defined but that `window.aptos.account` is not a function. I'm not sure why that would be the case but the current implementation will not handle it; it needs to be `window.aptos?.account?.()`. Potentially all of the window.aptos method calls need to be changed in this way.

Reply:
it looks like the exception in this error is `TypeError: Cannot read properties of null (reading 'disconnect')` - which is a bit weird because we dont have `disconnect` in our app. 
Not completely sure what causes `window.aptos.account is not a function` but I guess to be safe we can add this check also